src/ontology/cl-edit.owl

```diff
@@ -9016,9 +9017,9 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000653 "podocyte"^^xsd:stri
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0000653 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:comment obo:CL_0000653 "GO has podocyte as a related synonym due to concern this may refer to more than one cell type. However, this did not turn up in my search. If podocyte does refer to more than one cell type, then we should classify the synonym to 'related'[tfm]."^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000653 "glomerular visceral epithelial cell"^^xsd:string)
+EquivalentClasses(obo:CL_0000653 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0006852)))
 SubClassOf(obo:CL_0000653 obo:CL_0002518)
```

Comment:
This classification should be inferable.  Please check by removing and re-running the reasoner.  Please delete if inferable. If not inferable we should look at the axiomatization of obo:CL_0002518 to ensure that it is.

Reply:
Thanks for taking a look, @dosumis. That equivalence axiom is not inferable right now. I could try again after obophenotype/uberon#2230 is done. Even still, wouldn't we have to leave in "subClassOf part of some visceral layer of glomerular capsule"? I do not see how the equivalence axiom would be inferred without it.

Reply:
@dosumis, thank you for looking at this.

The following is inferred: 
'kidney epithelial cell' and ('part of' some 'glomerular epithelium')

Should we also expect to see 'kidney epithelial cell' inferred under SubClass of? That is not explicitly inferred.